src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py

```diff
@@ -508,6 +508,204 @@ def test_functionapp_consumption_linux_powershell(self, resource_group, storage_
         self.cmd('functionapp config show -g {} -n {}'.format(resource_group, functionapp_name), checks=[
             JMESPathCheck('linuxFxVersion', 'PowerShell|7.2')])
 
+class FunctionAppManagedEnvironment(LiveScenarioTest):
```

Comment:
May I ask why do you make these test cases as `LiveScenarioTest`?

Reply:
We marked them as `LiveScenarioTest` because we've been testing the backend changes in stage and do not currently have a way to test in production regions

Reply:
OK, got it